styles.css

```diff
@@ -1,730 +1,538 @@
+img,
+main {
+  display: block;
+}
+
+li {
   list-style: none;
-  align-items: center;
-  justify-content: space-between;
-  margin-right: 0;
-  width: 286px;
-  height: 72px;
 }
 
-.nav a {
-  font-family: Inter, sans-serif;
-  font-style: normal;
-  font-weight: 600;
-  font-size: 15px;
-  line-height: 20px;
-  color: #fff;
+nav ul {
+  list-style: none;
+}
+
+a {
+  margin: 0;
+  padding: 0;
+  font-size: 100%;
+  vertical-align: baseline;
+  background: transparent;
   text-decoration: none;
 }
 
-.nav-list div {
-  margin: 12px 0 12px 24px;
-  font-family: Roboto, sans-serif;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 18px;
-  line-height: 24px;
-  height: 24px;
-  transition-property: margin;
-  transition-duration: 500ms;
+/* HTML5 Reset Stylesheets. */
+
+/* -----Header Section ---- */
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 1rem;
 }
 
 .logo {
-  font-family: Poppins, sans-serif;
-  color: #7f8cff;
-  font-size: 22px;
-  font-weight: 600;
-  line-height: 16px;
-  text-shadow: 2pc;
+  margin: 4px 199px 4px 12px;
+}
+
+.menu {
+  margin: 4px 24px;
 }
 
-/* -----------------nav bar-------- */
+nav {
+  display: none;
+}
 
-/* -------Presentation Section----- */
+/* -----Header Section ---- */
 
-section.main-container {
-  margin-bottom: 2rem;
-  display: grid;
+.wrapper {
+  position: relative;
+  background: #e5e5e5;
+  height: auto;
+  width: 1440px;
+  margin: 0 auto;
 }
 
-.main-container {
-  width: 100%;
+section.intro {
+  width: auto;
   height: auto;
   background-image: url("/img/header-shapes-mobile.png");
   background-size: cover;
   background-repeat: no-repeat;
   background-color: #fff;
   padding-bottom: 10px;
+  padding-top: 5px;
 }
 
-.head-2-container {
-  margin: 30px 20px;
-  width: 100%;
-}
-
-h1 .title {
+.title.description {
+  margin: 144px 24px 12px;
+  width: auto;
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
-  padding: 5rem 0 10px;
+  color: #172b4d;
 }
 
-p.description {
-  position: relative;
-  font-size: 15px;
-  color: #344563;
-  width: 295px;
-  height: 76px;
-  margin-top: 1rem;
-  margin-left: 0;
+h1.title {
+  margin-bottom: 12px;
 }
 
-.social-icons {
-  margin-top: 12px;
+p.description {
+  margin-bottom: 12px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
 }
 
-.social-icons > h3 {
-  text-transform: uppercase;
-  font-weight: 500;
-  color: #7f8cff;
-  padding-bottom: 15px;
+.social.container {
+  margin: 12px 24px;
+  height: auto;
 }
 
-.social-icons > ul {
+ul.social-icons {
   display: flex;
-  justify-content: flex-start;
-}
-
-.social-icons.ul {
-  list-style: none;
-  height: 40px;
-  margin: 0 auto;
+  margin-top: 12px;
   align-items: center;
-  padding: 2px;
 }
 
-.social-icons > img {
-  width: 20px;
-  height: 20px;
+ul.social-icons li img {
+  margin-right: 19.01px;
 }
 
-/* Styles Section 1 */
+/* "Works" section */
 
-.grid-container {
-  height: 552px;
+.works {
```

Comment:
- [ ] I think it's better to fix the styling for the mobile version of the works section. Your portfolio must be amazing for any size of the screen.
<img width="558" alt="Screen Shot 2022-08-27 at 11 08 19 PM" src="https://user-images.githubusercontent.com/47673311/187054479-663029a8-756d-4327-b598-03470c8fe155.png">